### PR TITLE
Renamed Component System to Entity Component System as per issue #22

### PR DESCRIPTION
**Pull request by danielcirket**
Hopefully haven't missed anything!

### CODE REVIEW

**Comment by danielcirket** [comment]
Oh crap, I failed!

**Comment by danielcirket** [comment]
I'm having a massive fail Monday!

**Comment by danielcirket** [comment]
Success!

File: CleanDevelopment.sln [hunk 1]
```diff
@@ -20,14 +20,14 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "docs", "docs", "{8715F244-D
 		README.md = README.md
 	EndProjectSection
 EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Component.Tests", "tests\Component.Tests\Component.Tests.xproj", "{69F5C049-59D2-448C-B009-8F1C48D537C5}"
-EndProject
-Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "Components", "src\Components\Components.xproj", "{FDB68596-4408-419D-87A0-A6A170DC8144}"
+Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "EntityComponentSystem", "src\Components\EntityComponentSystem.xproj", "{FDB68596-4408-419D-87A0-A6A170DC8144}"
```
Comment (smudge202) [inline CleanDevelopment.sln]: Could you also change the physical directories, please?

Comment (danielcirket) [inline CleanDevelopment.sln]: Oh crap, I'm really having a bad today, I'll do that now, I think I already did the tests but just not the main project.

That Monday feeling =[

**Comment by smudge202** [comment]
@danielcirket Remind me to show you how to sync remote repositories next time I stream. :)

**Comment by danielcirket** [comment]
@smudge202, sounds good, tbh I'm still a Git noob.

File: CleanDevelopment.sln (hunk 1 above)
Comment (smudge202) [inline CleanDevelopment.sln]: Don't feel bad. As simple as this task might have looked at first glance, there's a tonne of things to change in a tonne of different places. You were never going to spot them all, and even after review we'll probably have missed something. ;)

**Comment by danielcirket** [comment]
Hopefully I've not missed anything else and we can :shipit:

**Pull request merged**